docs/source/Plugin/P021_commands.repl

```diff
@@ -13,7 +13,22 @@
         Using this command, either from rules, via http or mqtt, the **Set Level** of the plugin can be changed. If the value differs from the currently set value, the settings are saved.
         "
         "
+        ``config,task,<taskname>,Hysteresis,<value|calculation>``
```

Comment:
We have `SetLevel`, so why not `SetHysteresis`?

Reply:
`SetLevel` already was a bit awkward, should have been `Level`, and `Hysteresis` is already quite a long word, so I thought about `SetHyst`, but that's also awkward... 🤯